packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -134,6 +154,19 @@ export class GuxAvatar {
     ) as JSX.Element;
   }
 
+  private renderCameraIcon(): JSX.Element {
+    if (this.changePhoto && this.size === 'large') {
+      return (
+        <gux-icon
+          class="gux-change-photo-icon"
+          icon-name="fa/camera-solid"
+          size="small"
+          decorative
```

Comment:
something off here. this is `decorative` but I don't see how a screen reader user will be informed about the option to change photo. There is no button etc... that will be read to the user.